main.js

```diff
@@ -439,7 +453,9 @@
         };
 
         items.forEach(function(item) {
-            itemQueue.push(item);
+            setTimeout(function(){
+                itemQueue.push(item);
+            }, 500); // have some healthy delay or steam will block you for flooding
         });
```

Comment:
It's already processing the queue while adding items to it, that's why it works.